pkg/apis/options/legacy_options.go

```diff
@@ -488,6 +488,8 @@ type LegacyProvider struct {
 	KeycloakGroups                         []string `flag:"keycloak-group" cfg:"keycloak_groups"`
 	AzureTenant                            string   `flag:"azure-tenant" cfg:"azure_tenant"`
 	AzureGraphGroupField                   string   `flag:"azure-graph-group-field" cfg:"azure_graph_group_field"`
+	MSEntraIDSkipGroupsFromGraph           bool     `flag:"ms-entra-id-skip-groups-from-graph" cfg:"ms_entra_id_skip_groups_from_graph"`
```

Comment:
I would just call them entra without the MS prefix
```suggestion
	EntraIDSkipGroupsFromGraph           bool     `flag:"entra-id-skip-groups-from-graph" cfg:"ms_entra_id_skip_groups_from_graph"`
```